Test/implementationDivision/focus/focus.bpl

```diff
@@ -1,4 +1,4 @@
-// RUN: %parallel-boogie /printSplit:%t /errorTrace:0 "%s" > "%t"
+// RUN: %boogie /printSplit:- /errorTrace:0 "%s" > "%t"
```

Comment:
Whoops! :) Out of curiousity was the old behavior non-deterministic and the test flaky?

Reply:
Yes, at least in this PR it became flaky otherwise.